eZ/Publish/SPI/Persistence/Content/Handler.php

```diff
@@ -258,4 +258,13 @@ public function loadReverseRelations($destinationContentId, $type = null);
      * @return \eZ\Publish\SPI\Persistence\Content The published Content
      */
     public function publish($contentId, $versionNo, MetadataUpdateStruct $metaDataUpdateStruct);
+
+    /**
+     * Remove the specified translation from all the Versions of a Content Object.
+     *
+     * @param int $contentId
+     * @param string $languageCode language code of the translation
+     * @param string $mainLanguageCode Content Object main language code
```

Comment:
From puristic point of view this should not be here, as Persistence has enough to figure it out. _(gateway can still have it)_

But in case you disagree, we will at least need to better document this so we make it clear it HAS TO be the correct main language, this is not for changing main and that will lead to strange behavior if it is not the right one.